app/assets/stylesheets/application.css.scss

```diff
@@ -18,7 +18,7 @@
 @import "static/*";
 @import "cities/*";
 
-.jfdi_signin, .jfdi_signup, .forbes_index, .forbes_show {
+.jfdi_signin, .jfdi_signup, .index, .show {
   @extend .light-background;
```

Comment:
these are autogenerated classes by the views. When they were unique, it didn't matter as much, now they might be being applied in other places. Considering that these styles are pretty harmless (we want this background everywhere anyway), I'm not too worried. But still